Matrix.py

```diff
@@ -0,0 +1,120 @@
+class Matrix:
```

Comment:
use the google style docstring to document your code.
check out the link below for an example:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html